website/docs/docs/cloud/use-visual-editor.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Edit and create dbt modelsr" 
+id: use-visual-editor      
+sidebar_label: "Edit and create dbt models" 
+description: "Access and use the visual editor to create or edit dbt models through a visual, drag-and-drop experience inside of dbt Cloud." 
+---
+
+# Edit and create dbt models <Lifecycle status='beta'/> 
+
+<p style={{ color: '#717d7d', fontSize: '1.1em' }}>
+Access and use the dbt Cloud visual editor to create or edit dbt models through a visual, drag-and-drop experience. Use the built-in AI for custom code generation in your development experience.
+</p>
+
+:::tip Beta feature
+The visual editor provides users with a seamless and drag-and-drop experience inside of dbt Cloud. It's available in private beta for [dbt Cloud Enterprise accounts](https://www.getdbt.com/pricing). 
+
+To join the private beta, [register your interest](https://docs.google.com/forms/d/e/1FAIpQLScPjRGyrtgfmdY919Pf3kgqI5E95xxPXz-8JoVruw-L9jVtxg/viewform) or reach out to your account team to begin this process.
+:::
+
+## Prerequisites
+- You have an active [dbt Cloud Enterprise](https://www.getdbt.com/pricing) account.
+- You have a [developer license](/docs/cloud/manage-access/seats-and-users) ??? WHO can access editor? dev license? read-only?
```

Comment:
dev license required and dev credentials - yes?

Reply:
dev license
keep on latest

Reply:
can't create source model from this. need to have prod run that has sources created.